FWCore/Utilities/interface/std_pair_hasher.h

```diff
@@ -0,0 +1,19 @@
+#ifndef FWCore_Utilities_std_pair_hasher_h
+#define FWCore_Utilities_std_pair_hasher_h
```

Comment:
I'd rename the file `StdPairHasher.h' to be consistent with the struct name.

Reply:
OK